### PR TITLE
feat: Change default network name to Internal Node

### PR DESCRIPTION
**Pull request by yanguoyu**
1. Download and install Neuron for the first time. We can mock this to delete the `network.json` file at `/Users/{username}/Library/Application Support/Neuron/networks/index.json`

before:

image

after migrate:

image

2. Has added a new network but not edit the default network

before:

image

after migrate:

image

3. Has added a new network and edited the default network

before:

image

after migrate:

image

### CODE REVIEW

File: packages/neuron-ui/src/widgets/AlertDialog/index.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const AlertDialog = ({
           v === 'cancel' ? (
             <Button type="cancel" onClick={onCancel} label={t('common.cancel')} />
           ) : (
-            <Button type="confirm" onClick={onCancel || onOk} label={t('common.confirm')} />
+            <Button type="confirm" onClick={onOk || onCancel} label={t('common.confirm')} />
```
Comment (Keith-CY) [inline packages/neuron-ui/src/widgets/AlertDialog/index.tsx:54]: I haven't reviewed the previous PRs so I wonder why `onCancel` is bound to `confirm`

**Review by Keith-CY — APPROVED** [review]

File: packages/neuron-ui/src/widgets/AlertDialog/index.tsx (hunk 1 above)
Comment (yanguoyu) [reply packages/neuron-ui/src/widgets/AlertDialog/index.tsx:54]: When the type is `failed`, the `onCancel` function is used to close the dialog.

Comment (Keith-CY) [reply packages/neuron-ui/src/widgets/AlertDialog/index.tsx:54]: Got it

**Review by WhiteMinds — APPROVED** [review]

**Comment by silySuper** [comment]
/package.

**Comment by silySuper** [comment]
I need to verify that does the  internal node whether support edit  or not now?

**Comment by yanguoyu** [comment]
> I need to verify that does the internal node whether support edit or not now?

of cause

**Review by silySuper — APPROVED** [review]

**Pull request merged**